README.md

```diff
@@ -18,6 +18,11 @@ To install dependencies, run `poetry install`.
 
     python main.py https://node1.mainnet.hathor.network/
 
+Or you can use the Docker image
+
+    docker build -t tx-mining-service .
+    docker run -it --network="host" tx-mining-service <full-node-address:port> --address <mining-reward-address> --api-port 8080 --stratum-port 8000
```

Comment:
Since we are pushing to a public registry we could just skip the local build and use the public image.

Reply:
True. But maybe someone wants to make changes and run his/her customized bersion in Docker.

I think it's useful to have the instructions to build locally.

But I can add a line about using the public image.

Reply:
ADded